lib/fog/profitbricks/models/compute/server.rb

```diff
@@ -175,7 +176,7 @@ def list_volumes
           volumes = result.body['items'].each { |volume| volume['datacenter_id'] = datacenter_id }
           result.body['items'] = volumes
 
-          result.body
+          result.body['items']
```

Comment:
This is breaking the current behaviour. If we do follow this pattern we need to do a major bump on `fog-profitbricks` so other users that rely on this wont be surprised when they bump the version.

Whats the data returned in this call? Only `items` key?